deployment/sandbox-binary/flyte_sandbox_binary_helm_generated.yaml

```diff
@@ -115,6 +115,19 @@ data:
           v2_signing: false
           auth_type: iam
       container: my-organization-flyte-container
+  005-flyte-binary-config.yaml: |
+    flyte:
+      admin:
+        disableClusterResourceManager: false
+        disableScheduler: false
+        disabled: false
+        seedProjects:
+        - flytesnacks
```

Comment:
is this indentation incorrect? I'd have expected it to be two spaces to the right.

Everything else seems indented correctly 🤔

Reply:
LGTM!